### DISCUSSION TITLE
Intel Wifi 6 AX201, used in gen 2 intel e14 thinkpads (20TA)

### DISCUSSION THREAD

**Discussion by gr1mreaper2007** [vanilla-wiiu/vanilla#44] [Hardware Results]
Tested with fully updated linux mint with a 2nd gen e14 thinkpad (intel spec), on my wiiu it hangs at the "wii u menu" loading screen after syncing (not sure if it's just my wiiu). As of 11/07/24 you will have to run "sudo killall dhclient" before attempting to reconnect. Apart from that issue everything else should work fine.

**Comment by itsmattkc** [comment]
Just a note, the `killall dhclient` thing is a temporary regression specific to the prebuilt release (see #31) and not related to the hardware. Self-compiled builds generally don't exhibit this behavior.

**Comment by techmuse8** [comment]
Despite having the same card, the program doesn't seem to detect the console after inputting the symbols; On Linux Mint 21.3 Cinnamon, here's my log after entering the code;

```
INFO SUCCESSFULLY BOUND SOCKET 82 on PORT 51001
RECEIVED SYNC/CONNECT SIGNAL
USING WPA CONFIG: /root/.vanilla/vanilla_wpa_key.conf
SUBPROCESS Successfully initialized wpa_supplicant
WAITING FOR CTRL INTERFACE
UNKNOWN SCAN RESPONSE: FAIL
(RETRYING)
UNKNOWN SCAN RESPONSE: FAIL
(RETRYING)
UNKNOWN SCAN RESPONSE: FAIL
(RETRYING)
UNKNOWN SCAN RESPONSE: FAIL
(RETRYING)
<repeats until I press cancel>
```

Card info from dmesg:
``
Detected Intel(R) Wi-Fi 6 AX201 160MHz, REV=0x430
``

**Reply by yappinflower** [reply]
Same, on Linux Mint 22.3 GNOME on a ThinkPad T495 I get:
`TEMPORARILY SET wlp1s0 TO UNMANAGED
rfkill: WLAN soft blocked
UNKNOWN SCAN RESPONSE: FAIL
 (RETRYING)
UNKNOWN SCAN RESPONSE: FAIL
 (RETRYING)
UNKNOWN SCAN RESPONSE: FAIL
 (RETRYING)
`
until I cancel. 
`Detected Intel(R) Wireless-AC 9260 160MHz`

**Reply by itsmattkc** [reply]
If you run `sudo rfkill unblock wlan` before running Vanilla, does it then work?

**Reply by yappinflower** [reply]
After running `sudo rfkill unblock wlan` it works! Thanks, Matt!

**Reply by itsmattkc** [reply]
Thanks for the info, the latest build should call `rfkill unblock wlan` automatically on connection so you don't need to do it yourself.
